### ISSUE TITLE
Autocomplete widget for entity in multiselect mode - returns error once more than one entity selected

### ISSUE THREAD

**Issue by MarshallRJ** [shesha-io/shesha-framework#2178]
**Describe the bug**
When using the Autocomplete widget for an entity in mulitselect mode - The search query returns an error once more than one entity has already been selected.

The query is to the Entities GetAll eg:
_https://pd-tutorial-api.azurewebsites.net/api/services/app/Entities/GetAll_

here is an example of the payload:

```
skipCount: 0
maxResultCount: 1000
entityType: Shesha.Core.Person
properties: id _displayName
sorting: _displayName
filter: {"in":[{"var":"Id"},[{"id":"9392f2fd-147c-4003-8ed0-dc74960d8169","_displayName":"Bob Smith","_className":"Shesha.Core.Person"},{"id":"4e74ee74-451d-4a78-b03b-bf85998a0fc5","_displayName":"System Administrator","_className":"Shesha.Core.Person"},{"id":"91e45d56-e767-46b3-bc6e-630cec994b6e","_displayName":"","_className":"Shesha.Core.Person"}]]}
```
Here is the error returned

```json
{
    "result": null,
    "targetUrl": null,
    "success": false,
    "error": {
        "code": 0,
        "message": "Your request is not valid!",
        "details": "The following errors were detected during validation.\r\n - Error trying to resolve field 'personList'.\r\nInner Exception: Failed to parse expression\r\n",
        "validationErrors": [
            {
                "message": "Error trying to resolve field 'personList'.\r\nInner Exception: Failed to parse expression",
                "members": null
            }
        ]
    },
    "unAuthorizedRequest": false,
    "__abp": true
}
```

**To Reproduce**
1. Add an Autocomplete and set it to an entity
2. Set it to multiselect
3. Save the form and use the autocomlete
4. Once more than one entity is selected you should receive the error in the console.

I have configured a from on the tutorial site that replicates the issue:

https://tutorial.shesha.dev/dynamic/ShaCompanyName.ShaProjectName/aaa.form.test

**Expected behavior**
The search should filter and allow more enmities to be searched and selected

**Screenshots**
![image](https://github.com/user-attachments/assets/789a2fe4-e6b8-4d7c-828b-cc91cadfb540)

**Additional context**
The error seems to occur regardless of the entity type and is across multiple projects.

**Comment by MarshallRJ** [comment]
Update: This can be solved but setting the Autocomplete to use raw values:

![image](https://github.com/user-attachments/assets/75be1f64-4e0a-4d10-b68e-155493793577)

Closing the issue.

**Comment by MarshallRJ** [comment]
Update Reopening the issue: Use raw values solves the issue with the filter, but the binding to the model is now incorrect as now only Id's are returned and not the entity objects.

**Comment by MarshallRJ** [comment]
I have pushed a fix: #2187
